sauce/features/accounts/customFlagNames/index.js

```diff
@@ -0,0 +1,164 @@
+import Feature from 'core/feature';
+import * as toolkitHelper from 'helpers/toolkit';
+
+let flags;
+let redFlagLabel;
+let blueFlagLabel;
+let orangeFlagLabel;
+let yellowFlagLabel;
+let greenFlagLabel;
+let purpleFlagLabel;
+
+export default class CustomFlagNames extends Feature {
+  constructor() {
+    super();
+    if (!toolkitHelper.getToolkitStorageKey('flags')) {
+      this.storeDefaultFlags();
+    }
+    if (typeof flags === 'undefined') {
+      flags = JSON.parse(toolkitHelper.getToolkitStorageKey('flags'));
+      this.updateFlagLabels();
+    }
+  }
+
+  shouldInvoke() {
+    return toolkitHelper.getCurrentRouteName().indexOf('account') !== -1;
+  }
+
+  invoke() {
+    $('.ynab-grid-cell-flag .ynab-flag-red').parent().attr('title', redFlagLabel);
+    $('.ynab-grid-cell-flag .ynab-flag-blue').parent().attr('title', blueFlagLabel);
+    $('.ynab-grid-cell-flag .ynab-flag-orange').parent().attr('title', orangeFlagLabel);
+    $('.ynab-grid-cell-flag .ynab-flag-yellow').parent().attr('title', yellowFlagLabel);
+    $('.ynab-grid-cell-flag .ynab-flag-green').parent().attr('title', greenFlagLabel);
+    $('.ynab-grid-cell-flag .ynab-flag-purple').parent().attr('title', purpleFlagLabel);
+  }
+
+  observe(changedNodes) {
+    if (!this.shouldInvoke()) return;
+
+    if (changedNodes.has('ynab-flag-red') || changedNodes.has('ynab-flag-blue')
+      || changedNodes.has('ynab-flag-orange') || changedNodes.has('ynab-flag-yellow')
+      || changedNodes.has('ynab-flag-green') || changedNodes.has('ynab-flag-purple')) {
+      this.invoke();
+    }
+
+    if (changedNodes.has('ynab-u modal-popup modal-account-flags ember-view modal-overlay active')) {
+      $('.ynab-flag-red .label, .ynab-flag-red .label-bg').text(redFlagLabel);
+      $('.ynab-flag-blue .label, .ynab-flag-blue .label-bg').text(blueFlagLabel);
+      $('.ynab-flag-orange .label, .ynab-flag-orange .label-bg').text(orangeFlagLabel);
+      $('.ynab-flag-yellow .label, .ynab-flag-yellow .label-bg').text(yellowFlagLabel);
+      $('.ynab-flag-green .label, .ynab-flag-green .label-bg').text(greenFlagLabel);
+      $('.ynab-flag-purple .label, .ynab-flag-purple .label-bg').text(purpleFlagLabel);
+
+      $('.modal-account-flags .modal').css({ height: '22em' }).append(
+        $('<div>', { id: 'account-flags-actions' }).css({ padding: '0 .3em' }).append(
+          $('<button>', { id: 'flags-edit', class: 'button button-primary' }).append(
+            'Edit '
+          ).append($('<i>', { class: 'flaticon stroke compose-3' }).append('<!---->'))
```

Comment:
I suspect the `.append('<!---->')` got left in while debugging this bad boy. Fine to let it get merged in but might be nice to remove if it doesn't serve an ongoing purpose.